source/book/include/marlin/book/Selection.h

```diff
@@ -104,96 +146,46 @@ namespace marlin {
        *  @brief remove entry.
        *  @param itr iterator from entry which should be removed.
        */
-      void remove( const_iterator itr ) ;
+      void remove( const const_iterator &itr ) ;
```

Comment:
Pass it by values before not by const reference

Reply:
the copy is made when I pass it to std::vector at  [Selection.c:76](https://github.com/JBenda/MarlinMT/blob/2007029303e70626b9ea0207ed3b5ceb8765cbc5/source/book/src/Selection.cc#L76)